helpers/pyband/pyband/auth.py

```diff
@@ -121,7 +121,9 @@ def is_reporter(self, validator: str, reporter_pubkey: str) -> bool:
 
         :return: True if the reporter is a registered reporter of the validator.
         """
-        reporter = PublicKey.from_acc_bech32(reporter_pubkey).to_address().to_acc_bech32()
+        reporter = (
```

Comment:
let's agree on line length

Reply:
We agreed with 119 line length and set it up to our vscode already